src/sass/_detail_page.scss

```diff
@@ -0,0 +1,14 @@
+.detail-page {
+  margin-top: -$spv--medium;
+  padding-bottom: 0;
+}
+
+.instance-detail-title {
```

Comment:
It's good idea to have the shared content in a common scss file. This seems instance specific, maybe keep it in a dedicated file?